.flake8

```diff
@@ -0,0 +1,5 @@
+    [flake8]
+    ignore = E203, E266, E501, W503, F403, F401
```

Comment:
Pour info : 
- E203 : Whitespace before ':'
- E266 : Too many leading '#' for block comment
- E501 : Line too long
- W503 : Line break occurred before a binary operator (updated soon, see [this doc](https://www.flake8rules.com/rules/W503.html) 
- F403 : 'from module import *' used 
- F401 : Module imported but unused

Reply:
Pourquoi désactiver ces erreurs ? Je les trouve assez utiles.

Reply:
Bonne nouvelle, j'ai réussi à toutes les résoudre sauf la W503! ＼(＾O＾)／

Reply:
TROP COOL !  ＼(＾O＾)／